modules/apps/commerce/headless/headless-commerce/headless-commerce-delivery-cart-impl/src/main/java/com/liferay/headless/commerce/delivery/cart/internal/resource/v1_0/CartItemResourceImpl.java

```diff
@@ -106,7 +106,7 @@ public CartItem postCartItem(@NotNull Long cartId, CartItem cartItem)
 		ServiceContext serviceContext = _serviceContextHelper.getServiceContext(
 			commerceOrder.getGroupId());
 
-		_cpInstanceService.getCPInstance(cartItem.getSkuId());
```

Comment:
@andrea-ale-sbarra why this call is needed? there is no assignment

Reply:
hey @marco-leo it must be what remains of a previous implementation, I removed it now